lib/CMakeLists.txt

```diff
@@ -30,18 +30,17 @@ endif()
 
 # Libraries
 
-set(PONYC_GBENCHMARK_URL https://github.com/google/benchmark/archive/v1.7.1.tar.gz)
-
+set(PONYC_GBENCHMARK_URL https://github.com/google/benchmark/archive/refs/tags/v1.7.1.tar.gz)
```

Comment:
this change seems unrelated and should be rolled backed.

Reply:
Reverted.